src/main/java/me/zeroeightsix/kami/util/CoordUtil.java

```diff
@@ -0,0 +1,104 @@
+package me.zeroeightsix.kami.util;
+
+import net.minecraft.client.Minecraft;
+
+import me.zeroeightsix.kami.util.Coord;
+import java.io.*;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Scanner;
+
+import static me.zeroeightsix.kami.util.MessageSendHelper.sendChatMessage;
+
+/**
+ * @author dominikaaaa
+ * Created by dominikaaaa on 18/02/20
+ * Updated by wnuke on 14/04/20 and 26/04/20
+ */
+public class CoordUtil {
+    public static final String coordsLogFilename = "KAMIBlueCoords.txt";
+    public static int[] getCurrentCoord(boolean chunk) {
```

Comment:
Classes like Vec3d and Vec2i exist so that you dont have to use arrays to pass around vectors which is unsafe and dangerous